### ISSUE TITLE
Wiki Pages are missing information and navigation issue

### ISSUE THREAD

**Issue by HectorWon** [cBioPortal/cbioportal#8]
Two steps are missing from the wiki. Here are the two steps I copy from Hongxin's cBioPortal Development Environment Setup document:

Step 6. If no settings.xml exists under your maven folder, you need to create it under ~/.m2 folder. The content is
<?xml version="1.0"?>
<settings>
  <servers>
	<server>
      <id>settingsKey</id>
  	<username>[your mysql username]</username>
  	<password>[your mysql password]</password>
	</server>
	<server>
        <id>localhost</id>
        <username>[your tomcat username] </username>
        <password>[your tomcat password]</password>
	</server>
  </servers>
</settings>

Step 7. Create database schema cgds_test

Wiki pages plus these two steps are sufficient to help a user setup cBioPortal at local machine.

However, the wiki is not very easy to navigate. Here is an example of a user following the wiki to setup local environment:

1. Click the link "Setting up a cBioPortal (start here)" from home page, jump to "Setting up a cBioPortal" page.

2. Open "System Requirements" page, and follow the instruction to install tools like Tomcat and Maven.

3. Open "Database Setup" page, and follow the instruction to setup local database.

4. The user is asked to refer to "System Requirements" page again to setup tomcat. And from "System Requirements" page the user is asked to refer to "Software Configuration" page.

5. Go to "Building cBioPortal from Source" page to get the instruction to build the project. First two part of this page has the links to "System Requirements" and "Software Configuration" page.

...

To make the wiki more user-friendly, two issues can be avoided:

1. The same or similar content appears on multiple pages.

2. User is asked to refer to one part of a certain page, and after a few steps following other pages, is asked to refer to another part of the same page again.

Basically, I think we can improve the wiki by rearrange pages and reorder the content.

**Comment by ecerami** [comment]
I have rewritten / re-organized most of the wiki pages to make thing much more linear.  Still a lot of steps, but hopefully easier to follow now.  Take a look at:  https://github.com/cBioPortal/cbioportal/wiki, and let me know what you think.

**Comment by HectorWon** [comment]
Hi Ethan,

Thank you. Now it is much easier to follow.

Best regards,
Minxin

**Comment by ecerami** [comment]
Closing the issue.
